demo/sync/syncGlobalManager.js

```diff
@@ -20,6 +20,10 @@ function initSync() {
         return reject(error);
       }
       var cloudUrl = $fh.getCloudURL();
+
+      // decode the cloud app url if its encoded - required for keycloak
+      cloudUrl = decodeURIComponent(cloudUrl);
```

Comment:
This sounds like common problem for keycloak and we should wrap $fh.getCloudURL(); to resolve this globally.